public/js/render.js

```diff
@@ -31,7 +31,7 @@ var renderModule = function (error, eventsArray) {
     aElement.className = 'event-link';
 
     h3Element.innerHTML = event.en.name;
-    h4Element.innerHTML = event.startTime + ' To ' + event.endTime + ' At ' + event.placeId;
+    h4Element.innerHTML = new Date(event.startTime).toDateString() + '</br>' + new Date(new Date(event.startTime).getTime()).toLocaleTimeString([], {hour: '2-digit', minute: '2-digit'}) + ' - ' + new Date(new Date(event.endTime).getTime()).toLocaleTimeString([], {hour: '2-digit', minute: '2-digit'});
```

Comment:
Looks good, just a couple of things:
- The time isn't accurate, you need to set it to UTC time (click on an event to see the difference)
- It would be nice if the format matched that on the specific event view, i.e. 24hour, not 12hour